app/styles/components/ecosystem-icons.css

```
@@ -1,3 +1,13 @@
+.ecosystem-icons-container {
+  position: relative;
+}
+
+.ecosystem-icons-container .grid-dots {
+  position: absolute;
+  top: -40px;
+  right: 0px;
```

Looks like `stylelint` is asking for `right: 0` instead.

```suggestion
  right: 0;
```